sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -86,6 +86,25 @@
     return super.onOptionsItemSelected(item);
   }
 
+  private AirMapInterface createMapWithTouchDisabled(AirMapViewTypes type) {
+    if (type == AirMapViewTypes.NATIVE) {
+      AirGoogleMapOptions options = new AirGoogleMapOptions(new GoogleMapOptions());
+      options.setTouchEnabled(false);
+      AirMapInterface airMapInterface = mapViewBuilder.builder(type).withOptions(options).build();
+      return airMapInterface;
+    }
+
+    // case where it's not a native map, doesn't use AirGoogleMapOptions
```

Comment:
should we instead place this setter into the builder, so we don't have to wait until initialization to set touch enabled/disabled?